yb-voyager/cmd/analyzeSchema.go

```diff
@@ -469,6 +469,12 @@ func checkDDL(sqlInfoArr []sqlInfo, fpath string) {
 				"https://github.com/yugabyte/yb-voyager/issues/578", "Add all Partition columns to Primary Key", "TABLE", regMatch[2], sqlInfo.formattedStmt)
 				}
 			}
+		} else if strings.Contains(sqlInfo.stmt, "drop temporary table") {
```

Comment:
Should the comparison be case insensitive?

Reply:
Yes, Done.